src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -614,15 +622,38 @@ private static void Cleanup(HashSet<string> artifactsToCleanup)
             }
         }
 
+        private static void UpdateTitle(in StartedClock runsChronometer, int totalBenchmarkCount, int benchmarksToRunCount)
+        {
+            if (RuntimeInformation.IsWindows())
+            {
+                int executedBenchmarkCount = totalBenchmarkCount - benchmarksToRunCount;
+                TimeSpan fromNow = GetEstimatedFinishTime(runsChronometer, benchmarksToRunCount, executedBenchmarkCount);
+                string toFinish = string.Empty;
+
+                if (executedBenchmarkCount > 0)
+                {
+                    toFinish = $"- {(int)fromNow.TotalHours}h {fromNow.Minutes}m to finish";
+                }
+
+                Console.Title = $"{benchmarksToRunCount} Remaining {toFinish}";
```

Comment:
That doesn't really tell you the progress. You need a % or fraction like you had before.

Reply:
```cs
Console.Title = $"{benchmarksToRunCount}/{totalBenchmarkCount} Remaining {toFinish}";
```